### PR TITLE
Add version history for 5.7.3

### PR DESCRIPTION
**Pull request by dgault**
For a full list of included PRs see https://github.com/openmicroscopy/bioformats/milestone/60?closed=1

### CODE REVIEW

File: docs/sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
 
+5.7.3 (2018 January 11)
+-----------------------
+
+File format fixes and improvements:
+
+* TIFF
+  - fixed a NullPointerException when reading a TIFF file from the root system directory
+  - improved support for large images that are stored as a single uncompressed tile with
+    multiple interleaved channels.
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.rst]: Unneeded full stop here

**Comment by hflynn** [comment]
Ant build isn't looking happy so doesn't look like we'll be able to get this staged this morning

**Comment by hflynn** [comment]
Everything marked for this milestone looks to be included, assuming the open PRs get merged for the release

**Comment by dgault** [comment]
Thanks @hflynn, all PR's open or closed marked for 5.7.3 should be merged and ready for the release by tomorrow

**Comment by hflynn** [comment]
Re-building merge docs now for final check

**Comment by hflynn** [comment]
https://ci.openmicroscopy.org/view/Docs/job/BIOFORMATS-DEV-merge-docs/992/ includes this PR, this content renders as expected and the links all work. Good to merge

**Comment by hflynn** [comment]
N.B. I had to open #3037 to get the build green - if the link is still broken tomorrow, we'll need to merge that for the release

File: docs/sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,49 @@
...
+* Amira Mesh
+   - now supports ``Avizo`` in the file header in addition to the existing support for ``AmiraMesh``
+* Becker & Hickl SPCImage
+   - added a fix for IllegalArgumentException when reading files with compressed data
+* Zeiss-CZI
+   - fixed an IndexOutOfBoundsException when creating ROI objects
+
+Bug fixes and improvements:
+
+* removed unused target utils-formats-api from ant build
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: Given the importance and the nature of this I would almost classify it under the last category (`Build`)

File: docs/sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,49 @@
...
+* Memoizer updated to use UUID for generating unique memo file directories
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: Needs additional wording to specify the scope as it only affects the memoization automated tests - not the general memoization API.

File: docs/sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -1,6 +1,49 @@
...
+* detect and fix Findbugs' ``SBSC_USE_STRINGBUFFER_CONCATENATION`` using StringBuilder
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: I would move the first 2 items (dropped Ant target and memoization tests fixes) to the last section as they are mostly internal and keep only the generic findbugs fixes in this section

File: docs/sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -1,6 +1,49 @@
...
+   - added support in original metadata for the fields ``ISPG`` and ``Is data cube``
+* TillPhotonics TillVision
+   - directory listings for .pst files are now sorted
+* MetaMorph
+   - directory listings are now sorted during file initialization
+* Amira Mesh
+   - now supports ``Avizo`` in the file header in addition to the existing support for ``AmiraMesh``
+* Becker & Hickl SPCImage
+   - added a fix for IllegalArgumentException when reading files with compressed data
+* Zeiss-CZI
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.rst]: `Zeiss-CZI` -> `Zeiss CZI`

**Pull request merged**